core/base/document.php

```diff
@@ -475,7 +478,11 @@ protected function get_initial_config() {
 		];
 
 		if ( static::get_property( 'has_elements' ) ) {
-			$config['elements'] = $this->get_elements_raw_data( null, true );
+			$data = DataManager::instance()->run( 'editor/documents/elements', [
```

Comment:
The elements are already received from server, why to request them again?

Reply:
@matipojo, no one uses this??